.circleci/build.sh

```diff
@@ -50,8 +61,15 @@ hash cmake ninja
 GLOW_DIR=$PWD
 cd ${GLOW_DIR}
 mkdir build && cd build
-CMAKE_ARGS=("-DCMAKE_CXX_COMPILER=/usr/bin/clang++-8")
-CMAKE_ARGS+=("-DCMAKE_C_COMPILER=/usr/bin/clang-8")
+
+if [[ "${CIRCLE_JOB}" == "PYTORCH" ]]; then
+    CMAKE_ARGS=("-DCMAKE_CXX_COMPILER=/usr/bin/clang++-7")
```

Comment:
I know you mentioned this but why do we need clang 7 for pytorch?

Reply:
Actually we dont. It is because we need a python3-clang7.0+ docker image, and we dont have python3-clang8 image......

Reply:
what stops from making one?

Reply:
> what stops from making one?

nothing, will do this tomorrow.